### DISCUSSION TITLE
Real-time data?

### DISCUSSION THREAD

**Discussion by ichen6** [grafana/grafana-infinity-datasource#575] [Q&A]
Is there a way to have a CSV file that is updated in realtime and have Grafana Infinity take in that data as it is updated?

**Comment by yesoreyeram** [comment]
What do you mean realtime CSV?

**Reply by ichen6** [reply]
Like if the CSV file is updated with new data. Is it possible to have Grafana Infinity re-read that file every few seconds to have new live data from that CSV file?

**Reply by yesoreyeram** [reply]
It can't read only the new line. It only have the capability to read the file as whole. ( No streaming capabilities yet ).

It can read the file every few seconds if you set the dashboard to auto refresh. But everytime it will read the whole file

**Reply by ichen6** [reply]
I see. Thanks for the suggestion, but it doesn't seem like it re-reads the file with auto-refresh? Unless I am doing something wrong.

**Reply by ichen6** [reply]
I do inline data and upload a csv file. That csv file will be updated with information. I set the grafana dashboard to auto-refresh, but the infinity datasource does not re-read the file, and only has the non-updated information. Do you have any advice for what I am trying to achieve?

**Reply by yesoreyeram** [reply]
Inline won't reread. That's a static data
